### PR TITLE
Set proctitle early

### PR DESCRIPTION
**Pull request by emulbreh**

### CODE REVIEW

**Comment by stefano-dh** [comment]
:+1:  if travis

File: lymph/cli/service.py [hunk 1]
```diff
@@ -77,6 +77,10 @@ def run(self):
 
         self._register_signals()
 
+        # set process title early since start() may take some time,
+        # even though endpoint may not be known yet
+        self._set_process_title()
+
         self.container.start(register=not self.args.get('--isolated', False))
 
         self._set_process_title()
```
Comment (eguven) [inline lymph/cli/service.py]: forgot to remove?

Comment (emulbreh) [inline lymph/cli/service.py]: proctitle currently contains the rpc endpoint, which is only known after start(). So we just call it again to replace the ``endpoint=None`` we got from the first call.

Comment (eguven) [inline lymph/cli/service.py]: :+1:

**Pull request merged**